src/main/integrations/electron-breadcrumbs.ts

```diff
@@ -202,4 +142,57 @@ export class ElectronBreadcrumbs implements Integration {
       return emit(event, ...args);
     };
   }
-}
+
+  return {
+    name: INTEGRATION_NAME,
+    setup(client: NodeClient) {
+      const clientOptions = client.getOptions() as ElectronMainOptions | undefined;
+
+      trackRendererProperties();
+
+      whenAppReady.then(
+        () => {
+          // We can't access these until app 'ready'
+          if (options.screen) {
+            patchEventEmitter(screen, 'screen', options.screen);
+          }
+
+          if (options.powerMonitor) {
+            patchEventEmitter(powerMonitor, 'powerMonitor', options.powerMonitor);
```

Comment:
just to be sure, I don't really know how exactly that works for Electron, but is it a problem if we patch this multiple times? Can there even be multiple clients in electron?

Reply:
👍